sailing-yacht-research-foundation/authentication-front-end
# Feature/sns 328

Task:
* [[Webapp] Connect to the streaming server to receive live data](https://scopicsoftware.atlassian.net/jira/core/projects/SNS/board?issue-key=SNS-328)

Now, this streaming playback only use competition unit id.
Step to test the streaming playback would be:
1. Make sure the env variable of the app is updated. I added streaming server over there.
2. Go to `host/playback?competitionUnitId=_THE_COMPETITION_UNIT_ID_`. You may use any competition unit id to test, but just to make it simple and quick, use this one `bd80854f-802b-4bbe-a9cf-fba6d6f13ba6`. 
3. Stream the data using `live-data-server`. If you use the competition unit id that mentioned above, just to make it quick, use the setup below. 
```
node src/test/race_simulation/simulateRace.js b59bb461-e6d6-4b45-95c4-646788c775b6 bd80854f-802b-4bbe-a9cf-fba6d6f13ba6 eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjgxYWQ2Nzg4LTQ1NDEtNDMxMi05YTdkLTQ5MmNhOGI4MTdiYSIsImlzc3VlZEF0IjoxNjI5MDg3OTI5MjAyLCJpYXQiOjE2MjkwODc5Mjl9.ObGzXCy0bHY60ofeB1Bl2n6okn95KJnBC9anDoxupzI
```
